onelogin_aws_cli/tests/test_configurationFile.py

```diff
@@ -0,0 +1,78 @@
+from io import StringIO
+from unittest import TestCase
+from unittest.mock import patch
+
+from onelogin_aws_cli.configuration import ConfigurationFile
+
+
+class TestConfigurationFile(TestCase):
+
+    def _helper_build_config(self, config_content: str):
+        str = StringIO()
+        str.write(config_content)
+        str.seek(0)
+        return ConfigurationFile(str)
+
+    def test_can_save_password_username_true(self):
+        cfg = self._helper_build_config("""[defaults]
+save_password = false
```

Comment:
This is still duplicated with xor1.  If you make this line true, you'll have covered all cases I believe =).